docs/configuration.md

```diff
@@ -54,13 +54,20 @@ custom_info: "PR-123"
 # - Request paramters (optional) can be included for HTTP destinations with the `params` field
 #   - if the report parameter is included, the report paramater would contain the salus report
 #   - when `report` is not included in params, the salus report will be located in the body of the request sent
-# The available formats are `json`, `yaml`, `txt` and `sarif`.
+# The available formats are `json`, `yaml`, `txt`, `sarif` and `cyclonedx-json`.
 # `verbose` is an optional key and defaults to false.
 # 
 # Each report hash can add post paramaters using the `post` key , 
 # - Salus reports can be sent as a report paramater by specifying the paramater name in `salus_report_param_name`
 # - additional post paramaters can be specified through the `additional_params` field
 #
+# Each report hash can also specify what http verb should be used (currently support `put` and `post` key),
+# and headers to set. 
+# - If not specified http verb defaults to `post`
+# - you can pass in `headers` or `headers_env_var` with the corresponding list of name and value pairs.
+# - Values specified in `headers_env_var` will be treated as an environment variable `ENV[val]` while values
```

Comment:
Salus has a pattern in place for ENV vars:
https://github.com/coinbase/salus/blob/8de754473f8efbc19574e04152a921c8a7644497/lib/salus/config.rb#L66
https://github.com/coinbase/salus/blob/8de754473f8efbc19574e04152a921c8a7644497/lib/salus/config.rb#L171-L187
https://github.com/coinbase/salus/blob/61505620672bdabb486e8454b9dd3867365a50da/spec/fixtures/config/envar_config.yaml#L2-L10

You should be able to leverage that here